monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/TypeFilter.tsx

```diff
@@ -54,9 +54,11 @@ const TypeFilter = ({allRows, setFilters} :TypeFilterProps) => {
   }
 
   return (
-    <SelectComponent placeholder={"Type"} options={typeFilters}
-                     selectedOption={selectedType} onChange={handleTypeChange}
-                     variant={SelectVariant.Standard}/>
+    <MonkeySelect options={typeFilters}
+                  setAllOption={true}
```

Comment:
So we're gonna add boolean values for all possible default values we might have? `setNoneOption`, `setEmptyOption` etc?

Reply:
No, it will support only this option. 
If you want to create a different default value, you'll have to pass it.
Besides, I never saw an Empty option.

Reply:
Empty option is used to indicate that the user needs to select something from the list. 
![image](https://github.com/guardicore/monkey/assets/36815064/f3b0aeaf-cb50-47a4-8078-c23db0743578)